src/features/game/types/images.ts

```diff
@@ -1045,18 +1050,22 @@ export const ITEM_DETAILS: Items = {
     image: sunflowerShield,
     description: "A hero of Sunflower Land. Free Sunflower Seeds!",
   },
-  "Reward 9": {
-    image: questionMark,
+  "Skull Hat": {
+    image: skullHat,
     description: "A reward for your war efforts",
   },
-  "Reward 10": {
-    image: questionMark,
-    description: "A reward for your war efforts",
+  "War Skull": {
+    image: skull,
+    description: "3 enemies gone",
   },
-  "Reward 11": {
-    image: questionMark,
+  "War Tombstone": {
+    image: warTombston,
     description: "A reward for your war efforts",
   },
+  "Undead Rooster": {
+    image: undeadChicken,
+    description: "A fallen solider during the war",
```

Comment:
`A fallen solider during the war`

Should be:

`A fallen soldier during the war`

Reply:
Nice catch 😄